source/DataAccess/DADGenerator.ts

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+import IntegrationConfigFactory from '../IntegrationConfigFactory';
+import { IIntegrationConfig, IntegrationType } from '../IIntegrationConfig';
+import { writeFile } from 'fs';
+
+export default class DataAccessDoc {
```

Comment:
Our standards are usually to make file names match the main export for the module. In this case, `DataAccessDoc`. `DAD` is a proprietary acronym, so I'd usually prefer to spell it out to increase readability.  Maybe compromise and make the name of the class and the file `DataAccessDocGenerator`?